src/core/auth/login.ts

```diff
@@ -1,35 +1,54 @@
 /**
  * @author TotalElderBerry (Brian Keith Lisondra)
  */
-
 import type {Response, Request} from "express"
 import { result } from "../../utils/response";
 import jwt from 'jsonwebtoken';
+import Student from "../../db/models/student"
+import bcrypt from 'bcrypt';
 
 
 export function login(request: Request, response: Response){
     switch(request.method){
         case 'GET':
```

Comment:
You can remove `GET` case here since only `POST` request will be used.

Reply:
noted with thanks sir. have a nice day!